gcode/src/lexer.rs

```diff
@@ -371,4 +404,44 @@ mod tests {
 
         assert_eq!(got.value, "+3.14");
     }
+
+    #[test]
+    fn two_multi() {
```

Comment:
It would probably be more readable if we rewrite the test to be something like this:

```rust
let expected = vec![
  ("G", 0),
  ("0", 0),
  ("X", 0),
  ...
];

let actual: Vec<_> = Lexer::new("...")
  .map(|tok| (tok.value, tok.span.line))
  .collect();

assert_eq!(actual, expected);
```

Reply:
For long g-code example tests, that's probably the way to go. Maybe we could create some helper methods that make testing easier. We're really missing these long-form g-code verification tests, so I took this one from the pull request @dr0ps made.

How about we keep this test as is for now, and rework these kinds of tests in the future?